internal/gcsx/random_reader_stretchr_test.go

```diff
@@ -642,12 +642,14 @@ func (t *RandomReaderStretchrTest) Test_ReadAt_MRDRead() {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func() {
+			var err error
```

Comment:
you dont need this seperate declaration since fakeMRDWrapper below is not a instance variable on the struct.